en/frames_plane/turbo_timber_evolution.md

```diff
@@ -0,0 +1,203 @@
+
+Key airframe features:
+
+- Spacious interior
+- Top battery hatch
+- Optional leading edge slats
+- Fowler flaps
+- Rugged landing gear with steering tailwheel
+- Exterior lighting pre installed
+- Optional floats
+- Gentle flying characteristics
+- Low drag with internal linkages and minimal protrusions
+
+The goal of this build was to create a platform which could be used for general PX4 testing/development.
+That design goal meant that naturally balanced controls representative of a "vanilla" airplane were desirable.
+Because classic RC planes are usually designed to be hand flown with no computer augmented flight controls, they tend to be specifically designed to be well trimmed and balanced out of the box.
+These airplanes also have more attention devoted to making sure they handle well in the air.
+While it's possible to fly even the most simple foamboard airplane, a lot of nuance in airborne handling can be finessed with a bit more engineering effort.
+This airplane is a premium example of that, with features like frise ailerons to minimize adverse yaw.
+
+Key Build Features
+* Easy overall build with minimal airframe setup
+* Easy access to Pixhawk USB and debug connector
+* FPV with camera pan mount
+* Air data provided by wing slung pitot static pod
+* Long flight times (with Liion battery option > 24 minutes)
+
+# Build
+
+## Parts List
+
+- [Turbo Timber Evolution PNP (includes motor, servos, esc, etc, all fully installed)](https://www.horizonhobby.com/product/turbo-timber-evolution-1.5m-pnp-includes-floats/EFL105275.html#)
+- [Pixhawk 4 Mini/GPS/Power module/ set](https://shop.holybro.com/pixhawk4-mini_p1120.html)
+- SIK telemetry radio
+- MS4525DO differential pressure module and pitot tube
+- [Caddx Vista FPV air unit](https://caddxfpv.com/products/caddx-vista-kit)
+- [DJI FPV Goggles](https://www.dji.com/fpv)
+- [ExpressLRS Matek Diversity RX](http://www.mateksys.com/?portfolio=elrs-r24)
+- [Custom designed 3D printed parts](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/fw/turbo_timber_evolution/3d_printed_parts.zip)
+  - Pixhawk 4 Mini mount and top GPS mount
+  - FPV pod and camera mount
+  - Pitot static pod and wing hardpoint hanger
+- Misc hardware: M3 hardware (standoffs, washers, bolts), XT30 connector, hot glue, heatshrink, Molex Microfit connectors
+- Silicone wiring (14awg for high current, 16awg for low current, 22awg for low power and signals)
+- 3.6Ah 4S LiPo OR 4s2p 18650 LiIon
+
+## Airframe Build
+
+The TTE comes out of the box very complete as-is.
+Servos and linkages have already been installed, and the only real work remaining is finishing work, such as installing the landing gear and horizontal stabilizer.
+For this portion of the assembly, I simply followed the manual.
+
+## FPV Pod
+
+![Window and front fuselage (hatch) with FPV Pod mounted on top](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod_hatch.jpg)

```

Comment:
I added the words here to make the image "accessible" - that means if someone has impaired vision they get a description of what the useful part of the image is showing.

Now I'm not sure how many such people will be using these docs but it is still useful because it helps you understand why you are showing the image.